subcommittees/mentoring/guidelines_for_checkout_discussion_sessions.md

```diff
@@ -0,0 +1,23 @@
+#Guidelines for checkout discussion sessions
+
+This document outlines some things that can be discussed during the checkout discussion sessions. The sessions are meant to be student driven, but some groups may be quieter than others. This is by no means an exhaustive list, other subjects are bound to come up. 
+
+## Beforehand:
+* Discussion lead creates etherpad. Post link to etherpad in the session list etherpad
+* Discussion lead creates hangout. Post link to hangout in the session list etherpad.
+
+## During:
+* Inform students that a summary of the discussion will be kept in the etherpad, and that ideas and issues may be culled for future use by the SCF.
+* Give an orientation of how the checkout procedure itself goes
+	* A signup etherpad will be sent out. Choose your subject, and sign up for a 
+session. Around 5 people gather on a hangout with one of the trainers. Each person will have 5 minutes to present a section of the material. The trainer will dictate where in the material the student should start. After presentation, there will be 5 minutes for
+	feedback from the trainer. All will be notified afterwards by email whether they
+	passed or not.
+	Tell people that it is often useful to have the material available on paper.
+* Make sure that all know where the lesson material can be found. Ensure that they all know about the instructor guide. 
+* Inform about known problems with the lesson (i.e. tell people that the python lesson is too long and that it is in the process of being cut). 
+* Go through each of the modules in the lesson. Ask if anybody have any problems, questions or comments about the module. Do anybody have thoughts about changes or additions or things that can be removed?
```

Comment:
I still suggest that we do this, even if just briefly, in case we have somebody on the quieter side who have a question to the material but who are not prone to interrupt.